demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -1027,14 +1027,22 @@ def test_content_release_identifier_exists():
     assert sha1, 'GIT_SHA1 path in config.yml has been chaged. Fix the demisto-sdk or revert changes in content repo.'
 
 
-@pytest.mark.parametrize('prev_ver, expected', [
-    ('v4.5.0', 'origin/v4.5.0'),
-    ('master', 'origin/master'),
-    ('20.13.0', 'origin/20.13.0'),
-    ('origin/master', 'origin/master'),
-    ('64cac0b349187b861c4c717951a634de52caba03', 'origin/64cac0b349187b861c4c717951a634de52caba03')
+@pytest.mark.parametrize('branch_name, prev_ver, expected', [
+    ('master', 'v4.5.0', 'origin/v4.5.0'),
+    ('master', 'master', 'origin/master'),
+    ('master', '20.13.0', 'origin/20.13.0'),
+    ('master', 'origin/master', 'origin/master'),
+    ('master', '64cac0b349187b861c4c717951a634de52caba03', 'origin/64cac0b349187b861c4c717951a634de52caba03'),
```

Comment:
`origin/64cac0b349187b861c4c717951a634de52caba03` will result in the bug you are trying to avoid.
this is not a case I would have added to the tests.